ui/ui-components/components/tabs/App.tsx

```diff
@@ -3,5 +3,14 @@ import { Models } from "../../utils/apiData";
 
 export default function AppTabs({ app }: { app: Models.AppType }) {
   const tabs = ["edit"];
-  return <Tabs name={app.name} draftType={app.type} tabs={tabs} />;
+
+  if (app.refreshQueryAvailable == true) {
```

Comment:
don't use `==` in javascript.  Either `===` or just `if (app.refreshQueryAvailable) {` because you know it's already a boolean